assets/js/amp-image-caption-toggle.js

```diff
@@ -0,0 +1,151 @@
+/* exported ampImageNoCaption */
+/* global lodash */
+/* eslint no-magic-numbers: [ "error", { "ignore": [ 0, -1 ] } ] */
+
+let ampImageCaptionToggle = ( function() { // eslint-disable-line no-unused-vars
```

Comment:
Shouldn't `ampImageCaptionToggle` be a `const`?

Reply:
Yep it should, I think I went through and replaced some `var` with `let` and did this one by mistake.